backend/app/services/background_chat_executor.py

```
@@ -0,0 +1,410 @@
+        self.db.add(assistant_subtask)
+        self.db.commit()
+
+        return task, user_subtask, assistant_subtask
+
+    def _get_default_model_config(self) -> Dict[str, Any]:
+        """Get default model configuration (from environment variables)."""
+        config_str = getattr(settings, "SUMMARY_MODEL_CONFIG", None)
+        if config_str:
+            try:
+                return json.loads(config_str)
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)